src/init.ts

```diff
@@ -152,13 +152,8 @@ async function generateTsConfig(options: Options): Promise<void> {
     }
   }
 
-  // typescript expects relative paths to start with './'.
-  const baseConfig = './' +
-      path.relative(
-          options.targetRootDir,
-          path.resolve(options.gtsRootDir, 'tsconfig-google.json'));
   const tsconfig = formatJson({
-    extends: baseConfig,
+    extends: './node_modules/gts/tsconfig-google.json',
```

Comment:
Should `path.join` instead of using `/` directly?

Reply:
I don't think we should have a different value here depending on on which platform the source code is generated, right?

Reply:
I'm not sure.  I would expect `tsconfig` to be like `package.json` where we can always use `/`, but we should probably make sure.

Reply:
Windows has always supported `/` as a path separator as well as `\`. So I don't see any issues with this if we consider all the current major OS's.

Reply:
Ok sounds good.